### PR TITLE
chore: remove validator and improve set base stake amount

### PR DESCRIPTION
**Pull request by linguists**

### CODE REVIEW

File: bin/commands/set-node-base-amounts.js [hunk 1]
```diff
@@ -49,7 +49,13 @@ exports.handler = async function (argv) {
     // use yoctoNEAR instead of NEAR in config to take into account staking rewards
     if (node.base != null && currentNodes[node.id].base_stake_amount != null
       && node.base.toString() !== currentNodes[node.id].base_stake_amount.toString()) {
-      nodesToUpdateBaseStakeAmount.push(node);        
+      const denom = NEAR.from(node.base !== "0" ? node.base : currentNodes[node.id].base_stake_amount);
+      const diff = NEAR.from(node.base).sub(NEAR.from(currentNodes[node.id].base_stake_amount)).abs();
+      const threshold = 0.05;
+      // Update base stake amount only if the diff is obvious. Ignore the difference due to staking rewards
+      if (NEAR.parse("1").mul(diff).div(denom).gt(NEAR.parse(threshold.toString()))) {
```
Comment (0xdefacto) [inline bin/commands/set-node-base-amounts.js:56]: why `NEAR.parse("1")`?

File: bin/commands/set-node-base-amounts.js [hunk 2]
```diff
@@ -49,7 +49,13 @@ exports.handler = async function (argv) {
...
+        nodesToUpdateBaseStakeAmount.push(node);
+      }
```
Comment (0xdefacto) [inline bin/commands/set-node-base-amounts.js:58]: Should we print a warning message hear?

Comment (linguists) [reply bin/commands/set-node-base-amounts.js:58]: There will be a summary for the caller to confirm before execution. So no extra warning is needed.

```bash
Nodes to update base stake amount:
[ { id: 'teoviteovi.poolv1.near', base: '0' } ]
? Confirm update? › (y/N)
```

File: bin/commands/set-node-base-amounts.js (hunk 1 above)
Comment (0xdefacto) [reply bin/commands/set-node-base-amounts.js:56]: I know, they are integers.

**Review by 0xdefacto — APPROVED** [review]

**Review by dongcool — APPROVED** [review]

**Pull request merged**